Modules/Filtering/ImageFilterBase/include/itkTernaryGeneratorImageFilter.hxx

```diff
@@ -0,0 +1,355 @@
+/*=========================================================================
+ *
+ *  Copyright NumFOCUS
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0.txt
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *=========================================================================*/
+#ifndef itkTernaryGeneratorImageFilter_hxx
+#define itkTernaryGeneratorImageFilter_hxx
+
+#include "itkTernaryGeneratorImageFilter.h"
+#include "itkImageScanlineIterator.h"
+#include "itkTotalProgressReporter.h"
+
+namespace itk
+{
+template <typename TInputImage1, typename TInputImage2, typename TInputImage3, typename TOutputImage>
+TernaryGeneratorImageFilter<TInputImage1, TInputImage2, TInputImage3, TOutputImage>::TernaryGeneratorImageFilter()
+{
+  this->SetNumberOfRequiredInputs(3);
+  this->InPlaceOff();
+  this->DynamicMultiThreadingOn();
+  this->ThreaderUpdateProgressOff();
+}
+
+/**
+ * Connect one of the operands for pixel-wise addition
```

Comment:
addition -> operation, same below

Reply:
Just removing Doxygen style comment in hxx file